routes/solutionsController.js

```diff
@@ -35,7 +35,7 @@ router.get(
     // });
 
     // if (valid_user) {
-    SolutionModel.find({ user_role: req.params.role, status: "Pending" })
+    SolutionModel.find({ user_role: req.params.role, $or:[ {'status':'Pending'}, {'status':'Active'}, {'status':'Accepted'}, {'status':'InActive'} ] })
```

Comment:
I specifically asked you to use the "Not" query, you still used or, please change it

Reply:
Not query not giving the desired result